src/test/java/com/mongodb/jdbc/MongoDatabaseMetaDataTest.java

```diff
@@ -383,15 +383,25 @@ void testGetSQLKeywords() throws SQLException {
 
     @Test
     void testGetNumericFunctions() throws SQLException {
-        final String expectedFunctions = "";
+        final String expectedFunctions =
```

Comment:
You don't need to have the function names repeated here. It is a little different than the ones from getFunctions which are giving details about the arguments, etc... Here, if a function name is in the list, it means that it support the Open Group CLI specification for this function.

Reply:
When I updated this to just have the names once, the unit test fails with
```
expected: <ABS,COS,DEGREES,FLOOR,MOD,RADIANS,ROUND,SIN,SQRT,TAN> but was: <ABS,ABS,ABS,ABS,COS,COS,COS,COS,DEGREES,DEGREES,DEGREES,DEGREES,FLOOR,FLOOR,FLOOR,FLOOR,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,RADIANS,RADIANS,RADIANS,RADIANS,ROUND,ROUND,ROUND,ROUND,ROUND,ROUND,ROUND,ROUND,SIN,SIN,SIN,SIN,SQRT,SQRT,SQRT,SQRT,TAN,TAN,TAN,TAN>
Expected :ABS,COS,DEGREES,FLOOR,MOD,RADIANS,ROUND,SIN,SQRT,TAN
Actual   :ABS,ABS,ABS,ABS,COS,COS,COS,COS,DEGREES,DEGREES,DEGREES,DEGREES,FLOOR,FLOOR,FLOOR,FLOOR,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,MOD,RADIANS,RADIANS,RADIANS,RADIANS,ROUND,ROUND,ROUND,ROUND,ROUND,ROUND,ROUND,ROUND,SIN,SIN,SIN,SIN,SQ ...
```

I suspect this is because we collect the function names [here](https://github.com/mongodb/mongo-jdbc-driver/blob/655fdf34c83ad00348bfdd7866599c9cdb0cf35e/src/main/java/com/mongodb/jdbc/MongoFunctions.java#L196) by grabbing each function in a given category. We just return that list (with duplicates) directly [here](https://github.com/mongodb/mongo-jdbc-driver/blob/655fdf34c83ad00348bfdd7866599c9cdb0cf35e/src/main/java/com/mongodb/jdbc/MongoDatabaseMetaData.java#L324), then in the test we use a simple `assertEquals` to compare.

We could avoid the duplicates by not marking each variant in `MongoFunctions.java` with the category -- maybe we can just mark the first? Or we can remove duplicates before stringifying the lists? The former idea could be a good option, but I suppose the latter is the safer choice since that means we won't lose information if we edit/delete some/all of the duplicates. What do you think?

Reply:
As discussed offline, removing duplicates before turning the list into a string is better/safer.

Reply:
resolved offline: went with option 2. We no longer add duplicates to the lists, so now the test has just one instance of each 👍